### ISSUE TITLE
Infinite respring loop.

### ISSUE THREAD

**Issue by soum91** [pwn20wndstuff/Undecimus#110]
I really wonder why no has submitted this issues or encountered it. But Springboard many a times fails to load causing to infinite loop. First time encountered this when I used resping flipswitch. That time I thought maybe it was an issue with the flipswitch and uninstalled if to avoid using it. But later this happened when I tried to escape safe mode. And to my utter surprise, it’s even occuring with ldrestart. This really needs to be taken into account.

**Comment by Cryptiiiic** [comment]
Post the crashlog of SpringBoard to pastebin. Then reply with a link here. @soum91

**Comment by soum91** [comment]
@Cryptiiiic, after force rebooting I never saw recent SB crash report on CrashReporter. I guess SB fails to load rather than crashing, though I’m not sure since I’m novice. 
Moreover my crash log got cleaned up by os due low storage. So I can’t provide now, will post it next time it happens.
One thing I was wondering if it was being caused  due the fact that I unlaoded tipsd. Is that possible?

**Comment by Cryptiiiic** [comment]
Not sure why tips would have anything to do with it. Check the system crash logs instead of CrashReporter.(Open settings) Privacy > Analytics > Analytics Data. Check for SpringBoard, backboardd, etc.

**Comment by soum91** [comment]
@Cryptiiiic As I have said my crash logs got cleaned so I don’t have any crashlog of when the issue occurred last time. Should I provide latest logs that I currently have? And I have found someone else who is also facing the same just after jb succeeds and s/he has also post on Reddit. They can help too I guess. I have asked them and waiting for them to answer.

**Comment by Samgisaninja** [comment]
If you deleted your logs, make it crash again?

**Comment by Cryptiiiic** [comment]
@soum91 do what sam said. ^ I'm closing this in 48-72 hours if no replies come in.

**Comment by soum91** [comment]
@Cryptiiiic, Well I don’t know what was the trigger but it happened with respring, exiting safemode and even with ldrestart. But when it happened last time was 3 days 5 hours and interestingly this is the highest uptime I have experienced since my unc0ver installation and really wish I won’t have force reboot anytime soon.
The thing is I really doubt that the behavior would be registered as crash since every loop would end up in force reboot. I don’t know why I can’t I reproduce it after opening this issue. And all the previous crash log got removed, the oldest sb crash log I have is from 15th nov.
Certainly I am not the one who has experienced this. There are other people who have posted it. Even issue#105 is somewhat similar.
Since I can’t reproduce it, I’m closing the issue. If I ever face this again and can provide crash log I would open another issue if guys ask me to.
